packages/cascara/src/modules/DataSelect/DataSelect.js

```diff
@@ -60,13 +68,15 @@ const DataSelect = ({
           <option value={value}>{value}</option>
         )}
       </Input>
-    </label>
+    </>
```

Comment:
label tag to wrap input

Reply:
This module is now wrapped by the new `LabeledModule` 👍